indexs.js

```diff
@@ -1,110 +1,45 @@
+const express = require('express')
+const mongoose = require('mongoose')
+const User = require('./user');
+let url = "mongodb+srv://prateek:java1234@cluster0.vkgzh.mongodb.net/nodedb?retryWrites=true&w=majority"
```

Comment:
still no use of the environment file,
create a env file for credential/config data